infrastructure/main.tf

```diff
@@ -126,3 +126,44 @@ resource "azurerm_key_vault_secret" "POSTGRES_DATABASE" {
   value        = "${module.reform-blob-router-db.postgresql_database}"
 }
 # endregion
+
+# region reports secrets
+
+data "azurerm_key_vault_secret" "bulk_scan_reports_email_username" {
+  key_vault_id = "${data.azurerm_key_vault.bulk_scan_key_vault.id}"
+  name         = "reports-email-username"
+}
+
+data "azurerm_key_vault_secret" "bulk_scan_reports_email_password" {
+  key_vault_id = "${data.azurerm_key_vault.bulk_scan_key_vault.id}"
+  name         = "reports-email-password"
+}
+
+data "azurerm_key_vault_secret" "bulk_scan_reports_recipients" {
+  key_vault_id = "${data.azurerm_key_vault.bulk_scan_key_vault.id}"
+  name         = "reports-recipients"
+}
```

Comment:
Should we reuse the exact same secrets as bulk-scan? Even if the values are currently the same, should the secrets be the same?

Reply:
@lgonczar Yes, checked also with @doncem